pom.xml

```diff
@@ -73,7 +73,7 @@
         <maven-surefire-plugin.version>2.22.1</maven-surefire-plugin.version>
 
         <!-- spring -->
-        <spring-boot.version>2.1.8.RELEASE</spring-boot.version>
+        <spring-boot.version>2.2.2.RELEASE</spring-boot.version>
```

Comment:
please check release train for spring boot and cloud dependencies https://spring.io/projects/spring-cloud

Reply:
Done, thanks: upgraded Spring Cloud dependency to Hoxton.RELEASE